todo-app/docker-compose-dev.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.4"
+
+services:
+  server:
+    build:
+      context: server/
+      dockerfile: Dockerfile
+    container_name: server
+    #env_file: .env
+    volumes:
+      - ./server:/app/server
+    restart: always
+    command: sh -c "yarn && yarn start"
+    ports:
+      - "8080:8080"
+    networks:
+      - web
+  client:
+    build:
+      context: client/
+      dockerfile: Dockerfile
+    container_name: client
+    volumes:
+      - ./client:/app/client
+    restart: always
+    command: sh -c "yarn && yarn dev"
```

Comment:
@Shrinidhikulkarni7 in docker-compose.yml, we do `yarn && yarn build && yarn start` which builds and serves static files.